### ISSUE TITLE
Create action to "force delete" resource

### ISSUE THREAD

**Issue by aalves08** [rancher/elemental-ui#249]

**Comment by HoustonDad** [comment]
Yes please!! Can this be on any resource though, vs limiting it to machine objects? 

I also think this should be hidden under some Expert / Advanced options and not enabled by default. Someone will use this wrong and wreck their cluster, it's gonna happen.

**Comment by jitbasemartin** [comment]
What is the correct way to delete a machine in the Inventory ? The button in the UI is disabled

<img width="815" alt="Image" src="https://github.com/user-attachments/assets/25196758-3181-42ed-8e3f-e4299ea5158e" />

**Comment by aalves08** [comment]
@jitbasemartin you have to delete the property `metadata.finalizers`. You can do that via `Edit via YAML`
